tests/integration/test_db.py

```diff
@@ -288,6 +288,7 @@ async def test_sentry_db_blocked(ops_test: OpsTest, charm: str) -> None:
         )
 
 
+@pytest.mark.unstable
 async def test_weebl_db(ops_test: OpsTest, charm: str) -> None:
```

Comment:
A new weebl revision was released last week and it fails relating with our charms. Marking the test as unstable and created [DPE-2159](https://warthogs.atlassian.net/browse/DPE-2159) to investigate.